Dockerfile

```diff
@@ -19,11 +19,15 @@ FROM python:3.6-alpine
 MAINTAINER renku
 
 RUN apk update && \
-    apk add --no-cache curl && \
+    apk add --no-cache curl build-base libffi-dev openssl-dev && \
```

Comment:
I think some of these dependencies are actually not necessary and ended up weeding out the Dockerfile where you probably got them from (see https://github.com/SwissDataScienceCenter/renku-gateway/pull/43).

Reply:
They are for installing dependencies for one of the cryptography libraries I think. There is no wheel for alpine so they have to compile from source.